backend/backend/views/database_check_view.py

```diff
@@ -0,0 +1,8 @@
+from rest_framework import viewsets
+from backend.serializers.database_check_serializer import DatabaseCheckSerializer
+from backend.models.database_check_model import DatabaseCheckModel
+
+class DatabaseViewSet(viewsets.ModelViewSet):
+    # same as 'SELECT * FROM backend_databasecheckmodel;'

```

Comment:
If we leave the status page public, and this API is open, it's very easy to see how this query might become expensive than it needs to be.

What if we changed it to `DatabaseCheckModel.objects.count()`?

Also, this function doesn't return anything - is that on purpose? Should we return true?
```
if DatabaseCheckModel.objects.count() >=0
  return true
else
  return false
```

Reply:
Discussed offline about this one. I added some configurations to the API that only allows us to do GET requests, so we cannot add data through this API.

Also, yes this is normal that this function doesn't return anything (for now), that's on purpose.